README.md

```diff
@@ -26,8 +26,13 @@ users can know what their official response was.
 Send GitHub the issue URL at the end of the message so that they can
 more easily find updates and further comments here.
 
-Issues will only be closed once GitHub implements / fixes them,
-or explicitly says WONTFIX, which almost never happens nowadays.
+Issues will only be closed in one of the following situations:
+- once GitHub implements / fixes it
+- GitHub explicitly says it's a WONTFIX, which almost never happens nowadays
+- it's a duplicate of another issue
+- it contains a lot of abusive content
+
+However if an issue is just a stupid feature, it will likely get many 👎but it can be still opened.
```

Comment:
I think this patch makes sense. Except for this sentence:

`However if an issue is just a stupid feature, it will likely get many 👎but it can be still opened.`

I would entirely drop it. Why?

1. Even mentioning the question if something is "stupid" will likely just open up a can of worms of useless debates. 
2. The 4 rules already clearly state what is to be closed and what is not.

If you do keep it, I would suggest:

* comma after However
* write out thumbs down instead of or in addition to using emoji

Disclaimer: I am not a maintainer here, it's just my opinion.

Reply:
I agree this needs reworded to be less offensive, but the clarification should remain, as stupidity is often more controversial than abuse. 🤷‍♂️